meta/main.yml

```diff
@@ -24,11 +24,11 @@ galaxy_info:
     - ubuntu
 
 dependencies:
-  - src: kamaln7.swapfile
-    version: "0.4"
-    swapfile_size: "{{ sb_debian_base_swap_file_size }}"
+  - src: geerlingguy.swap
+    swap_file_size_mb: "{{ sb_debian_base_swap_file_size }}"
```

Comment:
@speedlight should we point to a specific version here?

Reply:
Yes @sestrella thanks for pointed out.